helpers/flags.js

```diff
@@ -1,16 +1,37 @@
+const knownFlags = new Map();
```

Comment:
`lms-core` has [a similar helper](https://github.com/BrightspaceUI/lms-core/blob/main/src/helpers/feature-flags.js) and it's a bit more robust and I think was intended to be the primary helper that consumers use. This one was just for core since it can't depend on `lms-core`.

So either we should be merging these helpers somehow, and/or `lms-core` will need to call into this code when it gets a request for a flag.